packages/askar/src/wallet/__tests__/AskarWallet.test.ts

```diff
@@ -62,14 +62,35 @@ describe('AskarWallet basic operations', () => {
     expect(nonce).toMatch(/[0-9]+/)
   })
 
-  test('Create ed25519 keypair', async () => {
+  test('Create ed25519 keypair from seed', async () => {
     await expect(
       askarWallet.createKey({ seed: '2103de41b4ae37e8e28586d84a342b67', keyType: KeyType.Ed25519 })
     ).resolves.toMatchObject({
       keyType: KeyType.Ed25519,
     })
   })
 
+  test('Create ed25519 keypair from secret bytes', async () => {
+    await expect(
+      askarWallet.createKey({
+        secretKey: TypedArrayEncoder.fromString('2103de41b4ae37e8e28586d84a342b67'),
```

Comment:
Wel, it was added as a direct replacement from seed so that´s why it came from a string, but might be better to use Buffer or Key as other methods from `Wallet`.

Reply:
Do we still want to make this change? Or maybe in another PR?

Reply:
Ah yes, I'll move seed to Buffer as well. I think it will be better to enforce the use of characters other than alphanumeric. After all we are in breaking time!